packages/waku/src/lib/renderers/html-renderer.ts

```diff
@@ -239,13 +240,17 @@ export const renderHtml = async (
       default: { createFromReadableStream },
     },
     { ServerRoot },
+    { ServerRouter },
```

Comment:
This is not what we want. **waku/router should work without changing the core.** (otherwise, other router integration like TanStack Router wouldn't be possible.)

You should change `getSsrConfig` in `unstable_defineRouter`. Let me know if you need any help.